src/index.html

```diff
@@ -1,13 +1,43 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div data-qa="card" class="product-card">
+      <img class="product-card__image" src="images/imac.jpeg" alt="picture">
+
+      <h1 class="product-card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h1>
+      <section class="product-card__code">Код товара: 195434</section>
```

Comment:
```suggestion
      <p class="product-card__code">Код товара: 195434</p>
```
Семантично тут більше підійшов би тег 'p', адже це просто текст, а не ціла секція